src/command_modules/azure-cli-network/azure/cli/command_modules/network/_help.py

```diff
@@ -1201,9 +1187,6 @@
     type: command
     short-summary: Create a network interface.
     examples:
-        - name: Create a network interface for a specified subnet on a specified virtual network.
-          text: >
-            az network nic create -g MyResourceGroup --vnet-name MyVnet --subnet MySubnet -n MyNic
```

Comment:
Same. This is not simply showing the required parameters.

Reply:
Fixed